cmd/util/util.go

```diff
@@ -14,12 +14,13 @@ import (
 
 const usage = `Usage:
 
-    $ util [-help] {-make-root|-make-intermediate|-make-dc} [-cert-in PATH] [-key-in PATH] [-out PATH] [-key-out PATH]
+    $ util [-help] {-make-root|-make-intermediate|-make-dc|-make-dcvectors} [-cert-in PATH] [-key-in PATH] [-out PATH] [-key-out PATH]
```

Comment:
I take it `make-dcvectors` should be removed?

Reply:
Oh, yes! A forgotten one here ;)